src/commands/LoadController.php

```diff
@@ -184,12 +184,35 @@ protected function process($data)
             }
         }
 
-        foreach (ArrayHelper::getValue($data, 'assign', []) as $userid => $permissionOrRoles) {
+        $assignData = ArrayHelper::getValue($data, 'assign', []);
+        $findBy = ArrayHelper::remove($assignData, 'findBy');
```

Comment:
Instead having a null value is the key doesn't exist you can provide the [default value arguments](https://www.yiiframework.com/doc/api/2.0/yii-helpers-basearrayhelper#remove()-detail) with [`findIdentity`](https://www.yiiframework.com/doc/api/2.0/yii-web-identityinterface#findIdentity()-detail)

So the following is null is not needed anymore

Reply:
Will do.